lib/spack/spack/platforms/cray.py

```diff
@@ -22,7 +22,9 @@
     'x86-naples': 'zen',
     'x86-rome': 'zen',  # Cheating because we have the wrong modules on rzcrayz
     'x86-skylake': 'skylake_avx512',
-    'mic-knl': 'mic_knl'
+    'mic-knl': 'mic_knl',
+    'interlagos': 'bulldozer',
+    'abudhabi': 'piledriver',
```

Comment:
We should try to find a more complete list of these. https://en.wikipedia.org/wiki/Opteron has some.

Reply:
What we really need is what name cray decided to give to it, which they haven't really been consistent about over time.

Reply:
I *think* most fo these should be aliases in archspec -- cray tends to name after the server part for AMD, and after the microarchitecture on intel, while we name after the microarchitecture in both places.  I think if we allow aliases we'll cover most of it and only need a few special cases like `mic-knl`.